libnetwork/osl/sandbox_linux_test.go

```diff
@@ -37,15 +37,18 @@ func generateRandomName(prefix string, size int) (string, error) {
 }
 
 func newKey(t *testing.T) (string, error) {
+	t.Helper()
 	name, err := generateRandomName("netns", 12)
 	if err != nil {
 		return "", err
 	}
 
 	name = filepath.Join("/tmp", name)
```

Comment:
Good point; if it can be, I should add that to https://github.com/moby/moby/pull/43346. I may have been conservative on that one, as it wasn't immediately clear to me if these can all be in separate directories, or if there's libnetwork code assuming they're all in the same path. (Need to check)